sound/soc/sof/sof-client-probes-ipc4.c

```diff
@@ -80,6 +80,14 @@ struct sof_ipc4_probe_point {
 
 #define INVALID_PIPELINE_ID      0xFF
 
+/**
```

Comment:
can we squash this with the previous patch that introduces the IP4 ops?

Reply:
Sure.